appveyor.yml

```diff
@@ -5,19 +5,13 @@ environment:
   DEPS_PREFIX: "C:/projects/nvim-deps/usr"
   # Silence/redirect errors due to missing locking support (via libgcov).
   GCOV_ERROR_FILE: "$(TEMP)/libgcov-errors.log"
+  # Add pip's user bin to PATH (python3).
+  PATH: "$(APPDATA)/Python/Python35/Scripts;$(PATH)"
```

Comment:
will this confuse the python2 tests? Also, I would put it at the end rather than the start.

Reply:
It is for `gcovr`.
But can use `python3 -m gcovr` for it, although that makes it bound to the version then really (i.e. `gcovr` would still work when switching to `python2 -m pip install gcovr` (although not without having both pip2 and pip3 in the PATH of course).